test/gurka/test_pbf_api.cc

```diff
@@ -70,9 +65,28 @@ TEST(pbf_api, pbf_in_out) {
       auto pbf_bytes = gurka::do_action(map, pbf_out);
       Api actual_pbf;
       EXPECT_TRUE(actual_pbf.ParseFromString(pbf_bytes));
-      // write_to_file("actual.pbf", actual_pbf.SerializeAsString());
-      // write_to_file("expected.pbf", expected_pbf.SerializeAsString());
       EXPECT_EQ(actual_pbf.trip().SerializeAsString(), expected_pbf.trip().SerializeAsString());
+      EXPECT_TRUE(actual_pbf.has_options());
```

Comment:
first i extended the existing testing to check that  in the default case all the fields come back, then i made sure that the setting of the field selector works by selecting only the trip part and checking that only it came back (except info which is non optional and except for the /status service which doesnt have a trip associated with it)